src/pages/RadicalCollaboration/Share/styles.js

```diff
@@ -0,0 +1,116 @@
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+const useStyles = makeStyles((theme) => ({
+  containerStyle: {
+    paddingTop: '96px',
+  },
+  imageStyle: {
+    border: '1px solid',
+    borderColor: theme.palette.background.darkGray,
```

Comment:
You could try condensing the border styling to one line with a template literal.
```suggestion
  imageStyle: {
    border: `1px solid ${theme.palette.background.darkGray}`,
```